internal/ptr/errors.go

```diff
@@ -0,0 +1,7 @@
+package ptr
+
+import (
+	"errors"
+)
+
+var JobNotFound = errors.New("job not found")
```

Comment:
I wanted to put this into the "jobs" package, but since we are using a pointer for a job in both "worker" and "jobs" packages, it is not possible (circular dependency - jobs package imports worker, so we can not import the error from the jobs package in the worker package). If you think of a better placement, feel free to recommend it.

Reply:
I would create `errors.go `under jobs, I don't see a problem when I checked locally. 
and if you need at to the worker as well and you have a cyclic import you can create an errors.go under worker as well.

Reply:
There is no need to share the error. Make a copy in each package.

If you ask me, creating a error value just to pass it into the logger is probably too much, I know the linter can complain so either disable it for that particular line or just make two copies.